build/analyze-bundle-size.mjs

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env node
+
+/**
+ * Bundle Size Analysis Script
+ *
+ * This script analyzes the bundle size of @microsoft/fast-element by creating
+ * production bundles using Vite and measuring their raw and gzipped sizes.
+ * Used by GitHub Actions and for local bundle size analysis.
+ */
+
+import { readFileSync, existsSync, writeFileSync, mkdirSync, rmSync } from "fs";
+import { resolve, join } from "path";
+import { gzipSync } from "zlib";
+import { build } from "vite";
+
+function formatSize(bytes) {
+    return `${(bytes / 1024).toFixed(2)} KB`;
+}
+
+async function createBundle(packageName, exportPath = ".") {
+    console.log(`\n🔨 Creating bundle for ${packageName} (export: ${exportPath})...`);
+
+    const tempDir = resolve("./.tmp/bundle-analysis");
+    const entryFile = join(tempDir, "entry.js");
+    const outputDir = join(tempDir, "dist");
+
+    try {
+        // Create temp directory
+        mkdirSync(tempDir, { recursive: true });
+
+        // Create entry file that imports FASTElement from the package
+        const entryContent = `import { FASTElement } from '${packageName}${
+            exportPath === "." ? "" : "/" + exportPath
+        }';
+export default FASTElement;`;
```

Comment:
This is only testing the import and export of one object (and side effects). Would `export * from "${packageName etc.}";` work?